securedrop/tests/functional/source_navigation_steps.py

```diff
@@ -123,6 +123,25 @@ def submit_page_loaded():
 
         self.wait_for(submit_page_loaded)
 
+    def _source_chooses_to_submit_documents_with_colliding_journalist_designation(self):
+        self.source_app.crypto_util.adjectives = self.source_app.crypto_util.adjectives[0]
```

Comment:
@kushaldas @rmol Using this does not affect the original list.

Directly importing and using source_app module shows:
```
    def _source_chooses_to_submit_documents_with_colliding_journalist_designation(self):
>       source_app.crypto_util.adjectives = source_app.crypto_util.adjectives[0]
E       AttributeError: module 'source_app' has no attribute 'crypto_util'
```

Reply:
The functional tests can be tricky. To see the error, the source interface server started by the functional test has to have the wordlists truncated. That's going to require changing `tests.functional.functional_test.FunctionalTest` to support customizing how the source app gets created in that server process, and then this test is going to need to use a subclass of `FunctionalTest` where that new method is used to abbreviate the wordlists in the source app's `CryptoUtil`. _Then_ the test should encounter a collision and see the error message.

If that much test work seems like the tail wagging the dog for this relatively straightforward change, I empathize, and would be happy to implement it so we can get your work merged.

Reply:
@rmol Oh ok. I get the point. The change does seem straightforward. Plus this a condition that might not ever be faced by anyone. Also, if the `flash` does fail, it will fail at many other places, which I guess would already have functional tests for this. But my views can be biased as I am the one who has to write this logic. Haha. So you peeps decide and let me know if a functional test for this is required are not.

Reply:
I think adding the test is a good idea, to make sure the error message is correctly shown upon collision. But you absolutely don't have to be the one adding it. We can take care of that.

Reply:
@rmol If it has to be added, I will give it a try once, and ping you peeps if I face any issue.